src/components/Modal/FolderModal/FolderModal.tsx

```diff
@@ -1,9 +1,17 @@
 import BaseModeal from "../BaseModal/BaseModal";
 import styles from "./folder.module.css";
 import { ADD_LINK, ADD_FOLDER, EDIT } from "utils/constants/strings";
+import { ModalProps } from "../BaseModal/BaseModal";
+import { GetFolderResponse } from "types/apis";
 
-function FolderModal({ variant, closeModal, link, list }) {
-  let title, action;
+interface Props extends ModalProps {
+  link?: string;
+  deleted?: string;
+  list?: GetFolderResponse[] | null;
```

Comment:
여기도 이런식으로 타입이 표현되어야 할 것 같아요~
```suggestion
  list?: Folder[] | null;
```